### DISCUSSION TITLE
Windows Store blocked on most corporate Domains - need direct download executable

### DISCUSSION THREAD

**Discussion by Geojim** [opengisch/QField#5843] [Ideas]
Would like to test the desktop side - to try sync files with our network drive - but yeh, no slick way to pull the files from qfieldcloud afaik.

**Comment by nirvn** [comment]
@Geojim , this makes little to no sense to me, sorry :) are you asking for a way to install QField on windows other that the windows store? If so, you can find windows installer in the release page (https://github.com/opengisch/QField/releases).

For the rest, you'd need to rephrase and expand on what you are looking for.
